R/grade_this.R

```diff
@@ -158,41 +316,44 @@ detect_grade_this <- function(env = parent.frame()) {
 #' # Call `debug_this()` inside your *-check chunk, is equivalent to
 #' debug_this()(submission)$message
 #' 
-#' # Or you can call `debug_this()` inside a `grade_this()` call
-#' # at the point where you want to get debug feedback.
-#' grade_this({
-#'   pass_if_equal(42, "Good stuff!")
-#' 
-#'   # Find out why this is failing??
-#'   debug_this()
-#' })(submission)
-#' 
-#' # Set default `fail()` message to show debug information
-#' # (for tutorial development only!)
-#' old_opts <- options(gradethis.fail = "{debug_this()}")
-#' 
-#' grade_this({
-#'   pass_if_equal(42, "Good stuff!")
-#' 
-#'   fail()
-#' })(submission)
-#' 
-#' # default fail() will show debug until you reset gradethis.fail option
-#' options(old_opts)
+#' # We won't run the remaining examples since they produce equivalent output
+#' if (interactive()) {
```

Comment:
Should this be a `\dontrun{`?
```suggestion
#' \dontrun{
```

Reply:
Or update the comment to say the remaining examples produce equivalent output